ert3/engine/_run.py

```diff
@@ -82,9 +85,17 @@ def _prepare_experiment_record(
     if record_source[0] == "storage":
         assert len(record_source) == 2
         ensemble_record = ert3.storage.get_ensemble_record(
-            workspace=workspace_root, record_name=record_source[1]
+            workspace=workspace_root,
+            record_name=record_source[1],
         )
 
+        # The blob record from the workspace has size one
```

Comment:
I think this is a good approach. This will be further improved in https://github.com/equinor/ert/issues/1919